mythril/analysis/modules/integer.py

```diff
@@ -168,20 +170,24 @@ def _handle_jumpi(self, state):
                 continue
             ostate = annotation.overflowing_state
             node = ostate.node
+
+            description_tail = "The binary {} operation can result in an integer overflow.\n".format(
+                annotation.operator
+            )
+
             issue = Issue(
                 contract=node.contract_name,
                 function_name=node.function_name,
                 address=ostate.get_current_instruction()["address"],
                 swc_id=INTEGER_OVERFLOW_AND_UNDERFLOW,
                 bytecode=ostate.environment.code.bytecode,
                 title="Integer Overflow",
-                _type="Warning",
+                severity="High",
+                description_head="The {} can overflow.".format(annotation.operator),
```

Comment:
The title and description head should be handled the same as how an SSTORE is handled

Reply:
Can you clarify that?